vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -462,7 +462,6 @@ def pt_weights_iterator(
         state = torch.load(bin_file, map_location="cpu", weights_only=True)
         yield from state.items()
         del state
-        torch.cuda.empty_cache()
```

Comment:
I think we can safely remove this `torch.cuda.empty_cache()`.

this line is introduced two years ago, in https://github.com/vllm-project/vllm/pull/784 . and the key is `del state` to reduce CPU memory footprint. `torch.cuda.empty_cache()` is not important here.